canonical-web-and-design/jaas-dashboard
# Replace Search and filter component

## Done

Replace Search and filter component using component from @canonical/react-components

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/models
- Click 'Search and filter' component in top right and click through all possible filters available
- Verify the models table filters accordingly
- Type a random word and press enter 
- Ensure only models with that word in their top level attributes appear
- Refresh your page and ensure the filters persist via query strings
- Test over multiple screen resolutions
- Test using a keyboard

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1467

## Screenshots

<img width="799" alt="Screenshot 2020-11-19 at 09 53 43" src="https://user-images.githubusercontent.com/505570/99650320-2bc34d00-2a4d-11eb-859d-f8fd99798d77.png">
